default.json

```diff
@@ -103,6 +103,16 @@
       "datasourceTemplate": "docker",
       "versioningTemplate": "docker"
     },
+    {
+      "description": "Update Github Action references in the README.md on renovatebot/github-action repository",
```

Comment:
The current setup seems to match any `README.md` files in any repository in our organization? Is that what we want?

Reply:
```suggestion
      "description": "Update Github Action references in the markdown files",
```